program-test/tests/cpi.rs

```diff
@@ -1,10 +1,9 @@
 use {
     solana_account_info::{next_account_info, AccountInfo},
-    solana_cpi::invoke,
     solana_instruction::{AccountMeta, Instruction},
     solana_keypair::Keypair,
     solana_msg::msg,
-    solana_program::instruction::get_stack_height,
+    solana_program::{instruction::get_stack_height, program::invoke},
```

Comment:
I think I saw you asking about this in Discord - confirming that reintroducing `solana_program::program::invoke()` is intentional as `solana_cpi::invoke()` is not 1-for-1 ?